lib/graphql/transaction.ts

```diff
@@ -91,6 +98,19 @@ export class Transaction extends IterableModel<TransactionModel> {
     return new ResultPage(this, transactions, pageInfo, args);
   }
 
+  /**
+   * Fetches first 50 matching transactions for a given user input
```

Comment:
`fetches up to first 50`?

Reply:
Good idea, c8d7e7dec33255d94d7e49a1b6c3d54ac3a484f9